docs/cvl/foundry-integration.md

```diff
@@ -0,0 +1,123 @@
+Foundry Integration (Alpha)
+=================
+
+Certora's Foundry Integration allows running foundry fuzz tests that exist for a project. Instead of executing the tests
+as fuzz tests for all inputs explicitly, the tests will be formally verified using the Certora Prover.
+This will yield higher guarantees of correctness as all inputs will be evaluated and tested. 
+
+While fuzzing is not as complete as formal verification (a fuzzer might “miss” some inputs that would expose a bug), 
+writing fuzz tests via Foundry is often easier for developer since it uses the same Solidity language as the smart contracts themselves.
+Foundry has tooling for writing fuzz tests on smart contracts, see [Foundry Documentation](https://book.getfoundry.sh/forge/fuzz-testing).
+
+This part of the documentation explains how to utilize the Certora Prover to formally verify tests that are written as Foundry fuzz tests.
+
+
+```{caution}
+This feature of the Certora Prover is in alpha state, so issues/unimplemented features are expected. Please contact us if you encounter 
+any issue.
+```
+
+# Usage
+
+There is a minimum of 2 required files to get the Prover to verify Foundry fuzz tests: A `.spec` file (written in CVL), and a `.conf` file. 
+
+- First, we need a `.spec` file written in CVL that will tell the Prover what to verify. The file is very simple, and in the minimal case 
+has exactly one line:
+
+```solidity
+use builtin rule verifyFoundryFuzzTests;
+```
+
+- Second, we need a `.conf` file that will provide the Prover the information of which contract to verify, and what spec file to use. 
+A minimal `.conf` file will look like this:
+
+```json
+{
+  "files": [
+    "path/to/file/with/Foundry/fuzz/tests.sol:<name_of_contract_containing_the_tests>",
+  ],
+  "verify": "name_of_contract_containing_the_tests:path/to/spec/file.spec",
+  "foundry_tests_mode": true,
+}
+```
+
+- Now, to run the tests, execute:
+`certoraRun path/to/conf/file.conf`
+- You will receive a link to a report containing the results of the run.
+
+For a full running example, please also see our [Example Repository](https://github.com/Certora/Examples/).
```

Comment:
Do we have a more specific example link we can give?

Reply:
```suggestion
For a full running example, please also see our [Foundry Integration Examples](https://github.com/Certora/Examples/tree/master/FoundryIntegration).
```